src/components-creators-insights/sq-modal-activation/sq-modal-activation.component.tsx

```
@@ -159,86 +172,97 @@ export default ({
       buttonClose={false}
       modalSize={width > 991 ? 'md' : 'lg'}
       forceMobileNoMargin={true}
+      id="modal-creators-insights"
     >
       <div className="box-insights">
         <p dangerouslySetInnerHTML={{ __html: messageModal || t('insightsText', { socialNetwork: 'instagram' }) }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)